lib/purview/types.rb

```diff
@@ -4,6 +4,7 @@
 require 'purview/types/date'
 require 'purview/types/float'
 require 'purview/types/integer'
+require 'purview/types/bigint'
```

Comment:
isn't there a typemap that also needs to be updated which maps a purview column type to a sql column type name?